### PR TITLE
Set page title to Event Description for View Event

### PR DESCRIPTION
**Pull request by ZhenyuYu1**
New Features:
1. Title and description metadata for page now set to the title and description of the event

File changes:
1. Moved contents of /view-event/page.tsx to ViewEvent.tsx to allow for dynamic metadata in /view-event/page.tsx.

### CODE REVIEW

File: src/components/ViewEvent.tsx [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+          id: eventId as UUID,
+          viewTime: new Date(),
+          title: data[0].title,
+          description: data[0].description,
+          starttime: data[0].starttime,
+          endtime: data[0].endtime,
+          timezone: data[0].timezone,
+          location: data[0].location,
+          config: data[0].config || {},
+          mode: data[0].mode || 'weekly', // TODO: need to modify to include both weekly & specific dates
```
Comment (pselvana) [inline src/components/ViewEvent.tsx]: Can you clean up this comment. We support this right?

**Comment by pselvana** [comment]
@ZhenyuYu1  - can you please test this out with a `wget` or `curl` locally of that view event page and seeing what the title is set to in the html returned.  If it is set to "loading...", then it won't help with previews like Slack link previews.

**Review by pselvana — APPROVED** [review]
LGTM after the export fix for edge function! I'd like to know if the wget shows you the real title or "loading"

File: src/components/ViewEvent.tsx (hunk 1 above)
Comment (ZhenyuYu1) [reply src/components/ViewEvent.tsx]: Removed comment.

**Pull request merged**